charts/spire/charts/spiffe-csi-driver/values.yaml

```diff
@@ -99,8 +99,8 @@ podSecurityContext: {}
 ## @param securityContext.privileged Flag for specifying privileged mode
 ##
 securityContext:
-  readOnlyRootFilesystem: true
-  privileged: true
+  # readOnlyRootFilesystem: true
+  # privileged: true
```

Comment:
I think we should either replicate the security context in the settings as it was before the change to make it configurable, or we should make it passthrough but set the defaults to what we had set before when we have recommendations set.

before we were doing:
```
            capabilities:
              drop:
                - all
            privileged: true
```

With the change if the user does nothing, they get instead:
```
{}
```